drivers/volume/portworx/portworx.go

```diff
@@ -4649,3 +4649,23 @@ func init() {
 	torpedovolume.Register(DriverName, provisioners, &portworx{})
 	torpedovolume.Register(PureDriverName, csiProvisionerOnly, &pure{portworx: portworx{}})
 }
+
+//Update the pool label for a particular pool id
```

Comment:
nit ( //Update)

Reply:
If you going to comment on what a function does, please follow the standards and use full finction name, example:
```
// UpdatePoolsLables updates pool labels for a given pool ID
func (d *portworx) UpdatePoolsLabels(n node.Node, poolID string, labels map[string]string) error {
}
```

Reply:
Changed as mentioned